### PR TITLE
DomQuery: fixed css2xpath for brackets in selector

### PR DESCRIPTION
**Pull request by milo**

### CODE REVIEW

File: src/Framework/DomQuery.php [hunk 1]
```diff
@@ -81,7 +81,16 @@ public static function css2xpath($css)
...
+					([~*^$]?)=(
+						(?:"[^"]*")|
+						(?:\'[^\']*\')|
+						(?:[^\]]+)
+					)
+				)?
+			\]|                                       # [attr=val] (3,4,5)
 			\s*([>,+~])\s*|                           # > , + ~ (6)
 			(\s+)|                                    # whitespace (7)
 			(\*)                                      # * (8)
```
Comment (milo) [inline src/Framework/DomQuery.php]: @dg I noticed that `$m[8]` is not used anywhere. Not sure how are the tests passing :)

File: src/Framework/DomQuery.php [hunk 2]
```diff
@@ -81,7 +81,16 @@ public static function css2xpath($css)
 		$xpath = '//*';
 		preg_match_all('/
 			([#.:]?)([a-z][a-z0-9_-]*)|               # id, class, pseudoclass (1,2)
-			\[([a-z0-9_-]+)(?:([~*^$]?)=([^\]]+))?\]| # [attr=val] (3,4,5)
+			\[
+				([a-z0-9_-]+)
+				(?:
+					([~*^$]?)=(
+						(?:"[^"]*")|
+						(?:\'[^\']*\')|
```
Comment (dg) [inline src/Framework/DomQuery.php]: Is (?: needed?

Comment (milo) [inline src/Framework/DomQuery.php]: Actully it's not. Would you prefer this multiline version or oneline only?

Comment (dg) [inline src/Framework/DomQuery.php]: Multiline is fine. Single line is write-only ;)

**Comment by milo** [comment]
Updated

**Comment by dg** [comment]
Gr8!

**Comment by milo** [comment]
Thanks for review.

**Pull request merged**